_faq/is-mentor-necessary.md

```diff
@@ -1,6 +1,6 @@
 ---
 question: "A showcased project on the website doesn't have a mentor, but
-	I'm interested in that project?"
+             I'm interested in that project?"
```

Comment:
"I'm .." on this line should be aligned exactly under "A showcase.." on the previous line.

i.e. remove two spaces.

Reply:
Still two spaces need to be removed